File: МДК 05.02/LR_1-2/3.4.pas
program hbhfc;
var a,b,c,m,n,x,d: real;
begin
  writeln ('введите 3 числа');
  read (a,b,c);
  d:=b**2-4*a*c;
  if d>0 then
    begin 
    m:=(-b+sqrt(d))/(2*a);
    n:=(-b-sqrt(d))/(2*a);
    writeln ('ответ: ',m,n);
    end;
  if d=0 then
    begin
    x:=(-b)/(2*a);
  writeln ('ответ: ', x);
  end
    else writeln ('ответ: корней нет');
end.

File: МДК 05.02/LR_1-2/3.4.py
a=int(input("введите a "))
b=int(input("введите b "))
c=int(input("введите c "))
d=b**2-4*a*c
from math import sqrt
if d>0:
    m=(-b+sqrt(d))/(2*a)
    n=(-b-sqrt(d))/(2*a)
    print("ответ: ",m,n)
if d==0:
    x=(-b)/(2*a)
    print("ответ: ",x)
else:
    print("ответ: корней нет")